next.config.js

```diff
@@ -10,6 +10,11 @@ module.exports = {
       issuer: /\.[jt]sx?$/,
       use: ['@svgr/webpack'],
     });
+    config.watchOptions = {
+      poll: true,
+      aggregateTimeout: 1000,
+    };
+
```

Comment:
What does this solve? NextJS should refresh automatically the page when there are UI changes.